lib/chewy/fields/base.rb

```diff
@@ -53,6 +55,18 @@ def compose(*objects)
 
     private
 
+      def field_with_ignore_blank_flag?(result)
+        @options[:ignore_blank] == true && field_with_empty_value?(result)
+      end
```

Comment:
This should work as well and is shorter

```suggestion
      def field_with_ignore_blank_flag?(result)
        @options[:ignore_blank] && result.blank?
      end
```